README.md

```diff
@@ -120,5 +120,19 @@ Run tests for a specific package or set of packages:
 PKGS=your,packages,here yarn test
 ```
 
+### Running the fullnode in Docker
+Running the fullnode in [Docker](https://www.docker.com/) allows us launch our entire stack with a single command. 
+
+To run the fullnode in Docker in production run:
+
+`docker-compose up`
+
+To run it in development run:
+
+```sh
+  docker-compose -f docker-compose.yml -f docker-compose.dev.yml run rollup-fullnode yarn
```

Comment:
Hmm I'm getting an error when running this command `ERROR: No such service: rollup-fullnode` (I ran it in the root directory)